include/config/keybind.h

```diff
@@ -18,6 +18,7 @@ struct keybind {
 	xkb_keycode_t keycodes[MAX_KEYCODES];
 	size_t keycodes_len;
 	int keycodes_layout;
+	bool allowWhenLocked;
```

Comment:
nit: we use pascal case in `rc.xml`, but use snake case for variable names.
```suggestion
	bool allow_when_locked;
```